testutils/anvil.go

```diff
@@ -40,3 +43,68 @@ func StartAnvilContainer(anvilStateFileName string) (testcontainers.Container, e
 		Started:          true,
 	})
 }
+
+type ContractAddresses struct {
+	ServiceManager         common.Address
+	RegistryCoordinator    common.Address
+	OperatorStateRetriever common.Address
+	DelegationManager      common.Address
+	Erc20MockStrategy      common.Address
+}
+
+func GetContractAddressesFromContractRegistry(ethHttpUrl string) (mockAvsContracts ContractAddresses) {
```

Comment:
This function reads all important contract addresses from the ContractsRegistry contract deployed. We can then use these addresses to build sdk clients.